server/src/test/java/org/elasticsearch/lucene/search/uhighlight/CustomPassageFormatterTests.java

```diff
@@ -21,7 +21,7 @@ public class CustomPassageFormatterTests extends ESTestCase {
     public void testSimpleFormat() {
         String content = "This is a really cool highlighter. Unified highlighter gives nice snippets back. No matches here.";
 
-        CustomPassageFormatter passageFormatter = new CustomPassageFormatter("<em>", "</em>", new DefaultEncoder());
+        CustomPassageFormatter passageFormatter = new CustomPassageFormatter("<em>", "</em>", new DefaultEncoder(), 0);
```

Comment:
Does it make sense to add context in the Test Class to this new argument used ?

Reply:
I could, but wouldn't going the class `CustomPassageFormatter` provide that context? I'm assuming you're suggesting adding context that the new parameter is for `numberOfFragments`? Correct me if I am wrong in my understanding your suggestion.